src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -12,36 +12,35 @@ import WMTS_Provider from 'Core/Commander/Providers/WMTS_Provider';
 import WMS_Provider from 'Core/Commander/Providers/WMS_Provider';
 import TileProvider from 'Core/Commander/Providers/TileProvider';
 import loadGpx from 'Core/Commander/Providers/GpxUtils';
-import GeoCoordinate,{UNIT} from 'Core/Geographic/GeoCoordinate';
+import GeoCoordinate, { UNIT } from 'Core/Geographic/GeoCoordinate';
 import Ellipsoid from 'Core/Math/Ellipsoid';
 import Projection from 'Core/Geographic/Projection';
 import CustomEvent from 'custom-event';
 import IoDriver_JSON from 'Core/Commander/Providers/IoDriver_JSON';
-import {STRATEGY_MIN_NETWORK_TRAFFIC} from 'Scene/LayerUpdateStrategy';
-
-var loaded = false;
-var eventLoaded = new CustomEvent('globe-loaded');
-var eventRange = new CustomEvent('rangeChanged');
-var eventCenter = new CustomEvent('centerchanged');
-var eventOrientation = new CustomEvent('orientationchanged');
-var eventLayerAdded = new CustomEvent('layeradded');
-var eventLayerRemoved = new CustomEvent('layerremoved');
-var eventLayerChanged = new CustomEvent('layerchanged');
-var eventLayerChangedVisible = new CustomEvent('layerchanged:visible');
-var eventLayerChangedOpacity = new CustomEvent('layerchanged:opacity');
-var eventLayerChangedIndex = new CustomEvent('layerchanged:index');
-var eventError = new CustomEvent('error');
-var JSONDriver = new IoDriver_JSON();
+import { STRATEGY_MIN_NETWORK_TRAFFIC } from 'Scene/LayerUpdateStrategy';
+
+let loaded = false;
+const eventLoaded = new CustomEvent('globe-loaded');
+const eventRange = new CustomEvent('rangeChanged');
+const eventCenter = new CustomEvent('centerchanged');
+const eventOrientation = new CustomEvent('orientationchanged');
+const eventLayerAdded = new CustomEvent('layeradded');
+const eventLayerRemoved = new CustomEvent('layerremoved');
+const eventLayerChanged = new CustomEvent('layerchanged');
+const eventLayerChangedVisible = new CustomEvent('layerchanged:visible');
+const eventLayerChangedOpacity = new CustomEvent('layerchanged:opacity');
```

Comment:
Do:
```js
this.viewerDiv.dispatchEvent(new CustomEvent('layeradded'));
```
or
```js
const LayerAddedEvent = 'layeradded'; // top-scope
[...]
this.viewerDiv.dispatchEvent(new CustomEvent(LayerAddedEvent)); // fn-scope
```
Instead of:
```js
var eventLayerAdded = new CustomEvent('layeradded'); // top-scope
[...]
this.viewerDiv.dispatchEvent(eventLayerAdded); // fn-scope
```